token-server/package.json

```diff
@@ -14,8 +14,8 @@
   "dependencies": {
     "@itwin/express-server": "^3.0.0",
     "@itwin/service-authorization": "^0.6.1",
-    "dotenv": "^8.2.0",
-    "dotenv-expand": "^5.1.0",
+    "dotenv": "^16.0.0",
+    "dotenv-expand": "^8.0.3",
```

Comment:
I believe after this upgrade, the [call](https://github.com/calebmshafer/itwin-viewer-proxy/blob/main/token-server/src/main.ts#L11) should be switched from
```javascript
  dotenv_expand(envResult);

```
to
```javascript
dotenv_expand.expand(envResult);
```
I would've loved to open a PR to fix it but I had no permissions to do so.

Reply:
Ah great, point. Thanks! Just opened #21 to fix the issue.